actioner/README.md

```diff
@@ -0,0 +1,34 @@
+# Actioner
+
+## Overview
+
+Actioner is a no-code app builder that lets you build your own apps for Slack and workflow automations without coding
+anything.
+This integration establishes necessary connection which lets you use Datadog API Actions in Actioner. Actioner does not
+publish or fetch data by itself, but a workflow in which Datadog API calls are made can be constructed.
```

Comment:
```suggestion
Actioner is a no-code app builder that allows you to build your own apps for Slack and workflow automations without requiring any coding.
This integration establishes the required connection to use Datadog API Actions in Actioner. Actioner does not
publish or fetch data, however, you can create a workflow to accomplish this using Datadog API calls.
```

Reply:
Quick question. When you refer to "workflow automations", are you referring to the Datadog product or to generic "workflows"? If you're referring to the Datadog product, can we change "workflow automations" to "Workflow Automation"?

Reply:
I'm referring generic workflows which may include Datadog API calls.